src/main/java/com/mappy/fpm/batches/tomtom/shapefiles/RoadShapefile.java

```diff
@@ -41,11 +48,13 @@ public void serialize(GeometrySerializer serializer, Feature feature) {
             LineString raw = geom(feature);
             LineString geom = isReversed(feature) ? (LineString) raw.reverse() : raw;
             Map<String, String> tags = roadTagger.tag(feature);
+            tags.put("source_pbf:country:tomtom", sourcePbfCountry);
```

Comment:
The PBF term would refer to the format of the OSM file.
Tomtom uses shapefiles instead, so the `_pbf` suffix doesn't seem accurate.

Moreover, one has to note that the country information isn't provided by Tomtom, but was named arbitrarily by the download job.

Reply:
Keeping current way of suffixing in mind, would `source:country:download_job` and `source:zone:tomtom` be better ?

Reply:
Not fan of the naming for `download_job`, but I don't have a better proposition.

Reply:
We could also remove semantic from the field, with something like `source:foldername` : 
- on the plus side it would be independant from the way the source shapefiles are organized (if I understand correctly, they might not be organized by country after all)
- on the minus side, this tag's name doesn't give much information, and might be easily misused.

Which one would you prefer ?

Reply:
As we use the semantics of the folder name to select which country we want to handle in our further processes, it would make sense to me to keep it in the naming.

Note: some folder do not directly point to a country (e.g. "Outerworld-France"). If we do not want the field to refer to a country, some synonyms of "region" may work : "area", "sector", "locality"